templates/people_admin/people_listing.html

```diff
@@ -0,0 +1,33 @@
+{% extends "public/components/base.html" %}
+{% load static %}
+
+{% block scripts %}
+  <script src="{% static "bundles/dashboards.js" %}"></script>
+{% endblock %}
+
+{% block content %}
+
+    	<section class="section">
+		<h1>People Overview</h1>
+
+		<div class="grid-x grid-margin-x">
+			<table class="hover">
+				<thead>
+                    <tr>
+                    	<th>State</th>
+                    	<th>Total Unmatched Entities</th>
+                    </tr>
+                </thead>
+                <tbody>
+					{% for state, spd in state_people_data.items %}
+						<tr class="row--align-top">
+                            <td>{{ spd.state }}</td>
+														<td><a href="{% url 'people_matcher' state %}">plenty</a></td>
```

Comment:
there was a hard coded URL reference here, which are fine/common enough, but when I moved the URL it broke the link, which was a good time to change this to a URL reference (it looks the link up by name here, so future changes in urls.py won't need to be reflected in the template)